package.json

```diff
@@ -40,7 +40,7 @@
     "ts-jest": "^26.5.3",
     "ts-node": "^9.1.1",
     "tslint": "^6.1.3",
-    "type-coverage": "^2.16.3",
+    "type-coverage": "^2.17.0",
```

Comment:
these are already in the `^` range, I'm a bit surprised the package.json file needed to be updated

Reply:
I personally want to keep package.json be latest too.

Reply:
Would it make sense to pin the dependency then? And use a tool like renovate to auto update it?

Reply:
`renovate` is already used at https://github.com/mdx-js/eslint-mdx/blob/master/package.json#L91-L95

I just used to run `yarn outdated` manually sometimes. 😂

Reply:
:thinking: If you want to automatically update lockfiles https://docs.renovatebot.com/configuration-options/#lockfilemaintenance could be added to the shared config

Reply:
It is enabled by extending `config:base` with `"rangeStrategy": "bump"`.

https://github.com/1stG/configs/blob/master/packages/renovate-config/package.json#L11-L23